README.md

```diff
@@ -56,60 +68,42 @@ without any changes, if you are just using one of the main classes and
 functions.
 
 ```js
-const { Readable, Writable, Transform, Duplex, pipeline, finished } = require('readable-stream')
-```
+const {
+  Readable,
+  Writable,
+  Transform,
+  Duplex,
+  pipeline,
+  finished
+} = require('readable-stream')
+````
 
 Note that `require('stream')` will return `Stream`, while
 `require('readable-stream')` will return `Readable`. We discourage using
 whatever is exported directly, but rather use one of the properties as
 shown in the example above.
 
-## Usage In Browsers
-
-You will need a bundler like [`browserify`](https://github.com/browserify/browserify#readme), [`webpack`](https://webpack.js.org/), [`parcel`](https://github.com/parcel-bundler/parcel#readme) or similar. With Webpack 5 (which unlike other bundlers does not polyfill Node.js core modules and globals like `process`) you will also need to:
-
-1. Install polyfills by running `npm install buffer process --save-dev`
-2. Create a [`webpack.config.js`](https://webpack.js.org/guides/getting-started/#using-a-configuration) file containing:
-
-```js
-const webpack = require('webpack')
-
-module.exports = {
-  plugins: [
-    new webpack.ProvidePlugin({
-      process: 'process/browser'
-    })
-  ],
-  resolve: {
-    fallback: {
-      buffer: require.resolve('buffer/')
-    }
-  }
-}
-```
-
```

Comment:
If this is no longer necessary (how?) then what is the purpose of https://github.com/nodejs/readable-stream/blob/main/test/browser/fixtures/webpack.browser.config.mjs?

Reply:
I made some mess with the readme, this is still needed, good spot.